### DISCUSSION TITLE
How to disable routePreloadOnLoad for specific routes?

### DISCUSSION THREAD

**Discussion by shobhitsinghal624** [umijs/umi#12428] [Q&A]
For a few routes, I need to disable this feature: [routePreloadOnLoad](https://github.com/umijs/umi/blob/master/packages/preset-umi/src/features/routePreloadOnLoad/routePreloadOnLoad.ts)

Please help with how to do that.

**Comment by fz6m** [comment]
This is an default feature and cannot be turned off through settings.

You can remove `package.json#name` to disable this feature, or you can modify the `dist/index.html` file to remove the part that preload script.
